From 5568599fa5ccaa650d399d43872f84efd807f02b Mon Sep 17 00:00:00 2001
From: Dennis Wenzel <b21659@freescale.com>
Date: Tue, 7 Dec 2010 15:44:03 -0600
Subject: [PATCH 199/505] ENGR00136072: mxc_epdc_x11_fb_test unit test build
 error

Changed code to reflect changes to mxcfb_update_data
data structure in linux/mxcfb.h.

Signed-off-by: Dennis Wenzel <b21659@freescale.com>
---
 test/mxc_x11_test/mxc_epdc_x11_fb_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_x11_test/mxc_epdc_x11_fb_test.c b/test/mxc_x11_test/mxc_epdc_x11_fb_test.c
index a144480..c49a4bb 100644
--- a/test/mxc_x11_test/mxc_epdc_x11_fb_test.c
+++ b/test/mxc_x11_test/mxc_epdc_x11_fb_test.c
@@ -175,7 +175,7 @@ int main (int argc, char* argv[])
 	mxcfbUpdateData.update_mode = UPDATE_MODE_FULL;
 	mxcfbUpdateData.waveform_mode = WAVEFORM_MODE_AUTO;
 	mxcfbUpdateData.temp = TEMP_USE_AMBIENT;
-	mxcfbUpdateData.use_alt_buffer = 0;
+	mxcfbUpdateData.flags = 0;
 	mxcfbUpdateData.update_marker = 0;
 
 	int numPanelUpdates = 0;
-- 
1.8.0

